spec/requests/admin/base_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::BaseController, type: [:request, :admin] do
```

Comment:
RSpec/FilePath: Spec path should end with `admin/base_controller*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)